Lake/Load/Manifest.lean

```diff
@@ -82,7 +78,7 @@ instance : ToJson Manifest := ⟨Manifest.toJson⟩
 protected def fromJson? (json : Json) : Except String Manifest := do
   let ver ← (← json.getObjVal? "version").getNat?
   match ver with
-  | 1 | 2 =>
+  | 3 =>
```

Comment:
We need to do *something* (other than throw "unknown version") for older manifest versions (1, 2). Since backwards compatible is likely infeasible, probably `` throw s!"incompatible manifest version `{v}`" ``.

Reply:
Okay, I've changed the word "unknown" to "incompatible".